src/README.md

```diff
@@ -4,16 +4,21 @@
 
 The Po.et node is made of micro-service like services.
 
+The services that make up the Po.et Node behave as if they were true microservices: isolated applications with no knowledge of each other's source code. In practice, they all live in the scame code base, are started by the root code file, `src/index.ts` and run in the same process, rather than as true separate applications.
```

Comment:
```suggestion
The services that make up the Po.et Node behave as if they were true microservices: isolated applications with no knowledge of each other's source code. In practice, they all live in the same code base, are started by the root code file, `src/index.ts` and run in the same process, rather than as true separate applications.
```